### PR TITLE
init Sentry

### PR DESCRIPTION
**Pull request by philipandersson**

### CODE REVIEW

File: apps/web/src/components/DataPill.tsx [hunk 1]
```diff
@@ -42,6 +43,7 @@ export const DataPill = ({
   }
 
   if (isStatus("error")) {
+    Sentry.captureException(query.error);
```
Comment (ericzakariasson) [inline apps/web/src/components/DataPill.tsx:46]: Will this trigger on every re-render?

**Review by ericzakariasson — APPROVED** [review]
Looks good, just one question! 🚀

File: apps/web/src/components/DataPill.tsx (hunk 1 above)
Comment (philipandersson) [reply apps/web/src/components/DataPill.tsx:46]: Yes, any issue with that? At least now with 100% sampling

Comment (philipandersson) [reply apps/web/src/components/DataPill.tsx]: `useEffect` to the rescue!

**Pull request merged**